docs/devguide/hexformat.rst

```diff
@@ -120,23 +120,12 @@ found here: `microbit-foundation/micropython-microbit-v2 <https://github.com/mic
 This is a port of MicroPython to the micro:bit which uses CODAL as the 
 underlying target platform.
 
-After cloning this repository update the submodules::
+Running the ``make`` command executes the following steps:
```

Comment:
There must be a step as well to add the layout able described in this page.

Reply:
@microbit-mark what I meant in this comment is to add something similar to V1 where it says that `tools/makeversionhdr.py` is invoked and stuff like that. You can have a look at the MicroPython Makefile, where all the steps should be listed. Let me know if you have any questions.

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/711/commits/939efde1f837cd9aea60e972e3bb135b215baff6 based on comments in https://github.com/microbit-foundation/micropython-microbit-v2/blob/master/src/Makefile

The final step listed is to `Build the MicroPython component`, but I am assuming this is part of the build process to create the hex. Is it a separate final step?

Reply:
I meant specifically info about this step:
https://github.com/microbit-foundation/micropython-microbit-v2/blob/76054c2e9445d8eea285132c2374a18ddc690514/src/Makefile#L39

Which is what generates and adds the layout table to the hex file, similarly to how the v1 sections lists the usuage of scrpts like  `tools/makeversionhdr.py` and `tools/adduicr.py`

Reply:
Thanks Carlos. I missed that. I've read through the process and updated the steps https://github.com/bbcmicrobit/micropython/pull/711/commits/5ee5691b4f91f8bf1a026e767e48853fad313f6c